foursight_core/deploy.py

```diff
@@ -147,18 +179,19 @@ def build_config(cls, stage, identity=None, stack_name=None,
                 curr_stage_environ['CHECK_RUNNER'] = check_runner
 
         filename = cls.get_config_filepath()
-        print(''.join(['Writing: ', filename]))
+        PRINT(''.join(['Writing: ', filename]))
```

Comment:
Untested, but I think you want:
```suggestion
        PRINT(f'Writing: {filename}')
```

Reply:
Yeah, seems more straight-forward to me; I'd left it as I'd found it.